aten/src/ATen/native/TensorFactories.cpp

```diff
@@ -30,12 +30,12 @@ void window_function_checks(
       options.layout() != kSparse,
       function_name,
       " is not implemented for sparse types, got: ",
-      options.type().toString());
+      at::getType(options).toString());
```

Comment:
We could consider printing the whole tensor options here. i.e. just `options` (it has the proper `operator<<`)

Reply:
Yeah, actually, this will be way more informative than the type. Thumbs up from me.

Reply:
OK, so, unrelatedly, there are a bunch of places where we have `Tensor x`, and we print `x.type().toString()`. Should we instead print `x.options()` instead? I'm less sure about this one...

Reply:
Fixed in #11145